static/styles/dark.css

```diff
@@ -378,6 +378,14 @@ body.dark-mode #out-of-view-notification {
     border: 1px solid 1px solid hsl(144, 45%, 62%);
 }
 
+body.dark-mode .email_tooltip {
```

Comment:
This commit has a type on the commit message: "Ajust".  Please make sure your commit messages are spelled properly!